tests/utils.py

```diff
@@ -146,3 +148,64 @@ def error_occurs_only_once(error_message: str, full_error: str) -> bool:
         return False
 
     return full_error.count(error_message) == 1
+
+
+@contextmanager
+def create_temp_file(content: str, filename: str) -> Generator[Path, None, None]:
```

Comment:
Mooi!

Reply:
All credits to @tim-vd-aardweg :sunglasses: